src/commands/dog.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch');
```

Comment:
please add node-fetch to package.json

Reply:
yeah sorry about that, i already had it installed on my bot, one second